apps/docs/src/components/code/PopupLayout.tsx

```diff
@@ -46,7 +46,7 @@ export const PopupLayout = ({ scope, code }: PopupLayoutProps) => {
   };
 
   return (
-    <section className="relative mt-md rounded-round [&>*]:border-border">
+    <section className="relative border-1 -m-t-px -m-l-px">
```

Comment:
this syntax is invalid in tailwind (probably a bug)
```suggestion
    <section className="relative border-1 -mt-px -ml-px">
```